site/docs/components/_content-template/index.mdx

```diff
@@ -0,0 +1,208 @@
+<!-- Tool-specific (e.g. Storybook) imports & exports go here -->
+
+<!-- Metadata about the component -->
+
+export const componentMetaData = {
+  /*
+   *  [REQUIRED] URL to the relevant directory that contains the (main) source
+   *  code for this component
+   */
+  sourceCodeUrl:
+    "https://github.com/some-org/some-repo/tree/main/path/to/component-name/",
+  /*
+   *  [REQUIRED] Name of the NPM package this component is published in
+   */
+  npmPackage: "@some-org/some-package",
+  /*
+   * [REQUIRED] The version of the NPM package that this component first
+   * appeared in.
+   */
+  npmPackageSince: "1.2.3",
+  /*
+   * [OPTIONAL] A list of synonymous names that someone might search for when
+   * trying find this component. For example, a "Dialog" component might list
+   * "Modal" and "Prompt" as its alsoKnownAs names.
+   */
+  alsoKnownAs: ["Other Name", "Another Name", "Different Name"],
+  /*
+   * [OPTIONAL] A list of other components that are related to this component
+   * and the nature of that relationship.
+   */
+  relatedComponents: [
+    /*
+     * Each related component must list name & relationship:
+     */
+    {
+      name: "Other Component Name",
+      /*
+        Permitted values for "relationship" are:
+        - contains:  This component contains the other component (e.g. Dropdown
+                     _contains_ List)
+        - similarTo: This component is similar to the other component (e.g. a
+                     Switch _similarTo_ Checkbox)
+      */
+      relationship: "contains",
+    },
+    {
+      name: "Another Component Name",
+      relationship: "similarTo",
+    },
+    // ...
+  ],
+};
+
+# Component Name
+
+<!--
+  Example screenshot or live component demo here of the default/most common
+  usage.
+
+  For screenshots use Markdown:
+
```

Comment:
I'm wondering if we should be more opinionated here and ask for live component, rather than image?

Reply:
Yeah, I agree. Will update the guidance accordingly.